must-gather/collection-scripts/gather_namespaced_resources

```diff
@@ -0,0 +1,94 @@
+commands_get+=("pods -owide")
+commands_get+=("nodes --show-labels")
+commands_get+=("pvc --all-namespaces")
+commands_get+=("subscription")
+commands_get+=("csv")
+commands_get+=("installplan")
+commands_get+=("events")
+commands_get+=("all")
+commands_get+=("role")
+commands_get+=("rolebinding")
+
+# collect oc output of OC desc commands
+commands_desc=()
+commands_desc+=("nodes")
+commands_desc+=("pods")
+commands_desc+=("storagecluster")
+
+# collect yaml output of OC commands
+oc_yamls=()
+oc_yamls+=("csv")
+oc_yamls+=("installplan")
+
+echo "collecting dump of namespace" | tee -a  "${BASE_COLLECTION_PATH}"/gather-debug.log
+oc adm --dest-dir="${BASE_COLLECTION_PATH}" inspect ns/"${install_namespace}" >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1
+echo "collecting dump of clusterresourceversion" | tee -a  "${BASE_COLLECTION_PATH}"/gather-debug.log
+for oc_yaml in "${oc_yamls[@]}"; do
+   # shellcheck disable=SC2129
+   oc adm --dest-dir="${BASE_COLLECTION_PATH}" inspect "${oc_yaml}" -n "${install_namespace}" >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1
+done
+
+# Run the Collection of Resources using must-gather
+for resource in "${resources[@]}"; do
+    echo "collecting dump of ${resource}" | tee -a  ${BASE_COLLECTION_PATH}/gather-debug.log
+    { oc adm --dest-dir=${BASE_COLLECTION_PATH}/namespaces/openshift-storage/ inspect "${resource}" --all-namespaces; } >> ${BASE_COLLECTION_PATH}/gather-debug.log 2>&1
+done
+
+# Create the dir for oc_output
+mkdir -p ${BASE_COLLECTION_PATH}/namespaces/openshift-storage/oc_output/
+
+# Run the Collection of Resources to list
+for command_get in "${commands_get[@]}"; do
+     echo "collecting oc command ${command_get}" | tee -a ${BASE_COLLECTION_PATH}/gather-debug.log
+     COMMAND_OUTPUT_FILE=${BASE_COLLECTION_PATH}/namespaces/openshift-storage/oc_output/${command_get// /_}
+     # shellcheck disable=SC2086
+     { oc get ${command_get} -n ${install_namespace}; } >> "${COMMAND_OUTPUT_FILE}"
+done
+
+# Run the Collection of OC desc commands
+for command_desc in "${commands_desc[@]}"; do
+     echo "collecting oc describe command ${command_desc}" | tee -a ${BASE_COLLECTION_PATH}/gather-debug.log
+     COMMAND_OUTPUT_FILE=${BASE_COLLECTION_PATH}/namespaces/openshift-storage/oc_output/${command_desc// /_}
+     # shellcheck disable=SC2086
+     { oc describe ${command_desc} -n ${install_namespace}; } >> "${COMMAND_OUTPUT_FILE}"
+done
+
+# NOTE: This is a temporary fix for collecting the storagecluster as we are not able to collect the storagecluster using the inspect command
+{ oc get storageclusters -n ${install_namespace} -o yaml; } > $BASE_COLLECTION_PATH/namespaces/openshift-storage/oc_output/storagecluster.yaml 2>&1
+
+# For pvc of all namesspaces 
+echo "collecting dump of oc get pvc all namespaces" | tee -a  ${BASE_COLLECTION_PATH}/gather-debug.log
+{ oc get pvc --all-namespaces; } >> ${BASE_COLLECTION_PATH}/namespaces/openshift-storage/oc_output/pvc_all_namespaces
```

Comment:
This shouldn't be saved under the `openshift-storage` namespace. Maybe create a new dir, `namespaces/all/`.